plugins/org/src/components/Cards/User/UserProfileCard/UserProfileCard.tsx

```diff
@@ -91,25 +90,23 @@ export const UserProfileCard = ({
             height="100%"
             width="100%"
           >
-            <Avatar
-              displayName={profile?.displayName}
-              picture={profile?.picture}
-            />
+            <Avatar displayName={displayName} picture={profile?.picture} />
           </Box>
         </Grid>
         <Grid item md={10} xl={11}>
-          <Typography variant="subtitle1">
-            <Box display="flex" alignItems="center">
-              <Tooltip title="Email">
-                <EmailIcon fontSize="inherit" />
-              </Tooltip>
-              {profile?.email && (
+          {profile?.email && (
```

Comment:
I changed the behavior here, if no email is available, we hide the row containing the email details completely